packages/strapi-generate-new/files/config/environments/development/security.json

```diff
@@ -49,7 +49,8 @@
       "Content-Type",
       "Authorization",
       "X-Frame-Options",
-      "Origin"
+      "Origin",
+      "X-Forwarded-Host"
```

Comment:
We only need to add it to the middleware configurations. So it should be removed.